src/bin/cobalt/serve.rs

```diff
@@ -30,6 +30,13 @@ pub fn serve_command_args() -> clap::App<'static, 'static> {
                  .help("Port to serve from")
                  .default_value("3000")
                  .takes_value(true))
+        .arg(clap::Arg::with_name("host")
+                 .short("H")
```

Comment:
Unsure whether there should be an abbreviation.

Cases when to have one
- The flag is used frequently enough that it is (1) easily assumed and (2) helping most people
- The abbreviation is a common convention

So far this seems like a less common case.  I don't have the familiarity with other applications to say there is a strong precedence for `-H`.  So I'm leaning against it.

Thoughts?

Reply:
yes sometime -H stands for human, i may use another (if any is good) or completely remove it.